crates/spfs-cli/cmd-monitor/src/cmd_monitor.rs

```diff
@@ -86,11 +89,10 @@ impl CmdMonitor {
         rt.block_on(self.wait_for_ready());
         // clean up this runtime and all other threads before detaching
         drop(rt);
-
+        #[cfg(unix)]
```

Comment:
Is this working for you? I think the monitor still needs to be background-ed to not block the runtime from launching, no?

Reply:
I guess you mean under windows?

So far i was able to trig the commands of course with error but not depending of the runtime blocking